src/datalad_installer.py

```diff
@@ -1063,6 +1071,9 @@ def provide(
             install_args: List[str] = []
             if batch:
                 install_args.append("--yes")
+            if channel is not None:
```

Comment:
I am curious if there is any reason to not have it as simpler

```suggestion
            if channel:
```

?

Reply:
It doesn't matter, and I prefer explicitly comparing against `None`.